marketplace/cmd/list.go

```diff
@@ -0,0 +1,89 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/dustin/go-humanize"
+	"github.com/google/go-github/v56/github"
+	"github.com/ignite/apps/marketplace/pkg/xgithub"
+	"github.com/ignite/cli/ignite/pkg/cliui"
+	"github.com/spf13/cobra"
+)
+
+const (
+	minStarsFlag     = "min-stars"
+	igniteAppTopic   = "ignite-cli-app"
```

Comment:
`ignite-cli-app` or `ignite-app`?

Reply:
[here](https://github.com/ignite/apps/pull/32/files#diff-4b73f00dc98389300755f39ae5356f68c63c5bf264de891d449cdbbf6104545eR70) is `ignite-app`. Maybe we should change to keep the consistency